site/content/en/docs/contributing/rest-api-design.md

```diff
@@ -1,8 +1,13 @@
-# REST API design principles
+---
```

Comment:
The changes in this file seem unrelated to the PR topic.

Reply:
Agree, but it is only one small file. Let's keep it in PR for simplicity.